lib/cylc/task_job_mgr.py

```diff
@@ -949,4 +948,5 @@ def _prep_submit_task_job_impl(self, suite, itask):
             'task_id': itask.identity,
             'try_num': itask.get_try_num(),
             'work_d': rtconfig['work sub-directory'],
+            'param_var': itask.tdef.param_var,
```

Comment:
The keys were sorted alphabetically. Please insert `param_var` in the right place.